modules/nw-ovn-k-anp-nbctl.adoc

```diff
@@ -0,0 +1,496 @@
+        port: 5353
+  - name: "allow-to-kapi-server" # rule1
+    action: "Allow"
+    to:
+    - nodes:
+        matchExpressions:
+        - key: node-role.kubernetes.io/control-plane
+          operator: Exists
+    ports:
+    - portNumber:
+        protocol: TCP
+        port: 6443
+  - name: "allow-to-splunk" # rule2
+    action: "Allow"
+    to:
+    - namespaces:
+        matchlabels:
+          tenant: splunk
+    ports:
+    - portNumber:
+        protocol: TCP
+        port: 8991
+    - portNumber:
+        protocol: TCP
+        port: 8992
+  - name: "allow-to-open-tenants-and-intranet-and-worker-nodes" # rule3
+    action: "Allow"
+    to:
+    - nodes: # worker-nodes
+        matchExpressions:
+        - key: node-role.kubernetes.io/worker
+          operator: Exists
+    - networks: # intranet
+      - 172.29.0.0/30
+      - 10.0.54.0/19
+      - 10.0.56.38/32
+      - 10.0.69.0/24
+    - namespaces: # open tenants
+        matchLabels:
+          tenant: open
+  - name: "pass-to-restricted-tenants" # rule4
+    action: "Pass"
+    to:
+    - namespaces: # restricted tenants
+        matchLabels:
+          tenant: restricted
+  - name: "default-deny"
+    action: "Deny"
+    to:
+    - networks:
+      - 0.0.0.0/0
+----
+====
+
+.Procedure
+
+. List pods with node information by running the following command:
++
+[source,terminal]
+----
+$ oc get pods -n openshift-ovn-kubernetes -owide
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                     READY   STATUS    RESTARTS   AGE   IP           NODE                                       NOMINATED NODE   READINESS GATES
+ovnkube-control-plane-5c95487779-8k9fd   2/2     Running   0          34m   10.0.0.5     ci-ln-0tv5gg2-72292-6sjw5-master-0         <none>           <none>
+ovnkube-control-plane-5c95487779-v2xn8   2/2     Running   0          34m   10.0.0.3     ci-ln-0tv5gg2-72292-6sjw5-master-1         <none>           <none>
+ovnkube-node-524dt                       8/8     Running   0          33m   10.0.0.4     ci-ln-0tv5gg2-72292-6sjw5-master-2         <none>           <none>
+ovnkube-node-gbwr9                       8/8     Running   0          24m   10.0.128.4   ci-ln-0tv5gg2-72292-6sjw5-worker-c-s9gqt   <none>           <none>
+ovnkube-node-h4fpx                       8/8     Running   0          33m   10.0.0.5     ci-ln-0tv5gg2-72292-6sjw5-master-0         <none>           <none>
+ovnkube-node-j4hzw                       8/8     Running   0          24m   10.0.128.2   ci-ln-0tv5gg2-72292-6sjw5-worker-a-hzbh5   <none>           <none>
+ovnkube-node-wdhgv                       8/8     Running   0          33m   10.0.0.3     ci-ln-0tv5gg2-72292-6sjw5-master-1         <none>           <none>
+ovnkube-node-wfncn                       8/8     Running   0          24m   10.0.128.3   ci-ln-0tv5gg2-72292-6sjw5-worker-b-5bb7f   <none>           <none>
+----
+
+. Navigate into a pod to look at the northbound database by running the following command:
```

Comment:
Why this specific ovnkube-node pod?

Reply:
No reason, I just picked one at random

Reply:
it is OK to pick any ovnkube-node, checked it on live cluster.